galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/ITsq.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.cicsts;
+
+import javax.validation.constraints.NotNull;
+public interface ITsq {
```

Comment:
It should extend the Comparable interface, so if you add two ITsq objects into a set<ITsq> you only end up with one instance ?
Solely based on the name of the queue I should think.

Reply:
HI @techcobweb , Yes you are correct. It will be 2 instances based on the queue name.  Please refer the [IVT program](https://github.com/Anuprakash-Moothedath/managers/blob/add_tsq_manager/galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.tsq.manager.ivt/src/main/java/dev/galasa/cicsts/tsq/manager/ivt/TsqManagerIVT.java).

```
	public ITsqFactory tsqFactoryA; 
	public ITsq tsqRecoverable;
	public ITsq tsqNonRecoverable;

...

         this.tsqFactoryA = cicsRegionA.getTsqFactory();
	// Create ITsq object for recoverable TSQ
	tsqRecoverable = this.tsqFactoryA.createQueue("GALASAR", true);
	// Create ITsq object for non-recoverable TSQ
	tsqNonRecoverable = this.tsqFactoryA.createQueue("GALASAN", false);
```